File: OLSource1/v1/VS_officedev/codesnippet/CSharp/customizing-a-ribbon-for-infopath_1.cs
        public string GetCustomUI(string ribbonID)
        {
            string ribbonXML = String.Empty;

            if (ribbonID == "Microsoft.InfoPath.Designer")
            {
                ribbonXML = GetResourceText("MyInfoPathProject.Ribbon.xml");
            }

            return ribbonXML;
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/customizing-a-ribbon-for-infopath_1.vb
    Public Function GetCustomUI(ByVal ribbonID As String) As String Implements Office.IRibbonExtensibility.GetCustomUI
        Dim ribbonXML As String = String.Empty

        If ribbonID = "Microsoft.InfoPath.Designer" Then
            ribbonXML = GetResourceText("MyInfoPathProject.Ribbon.xml")
        End If

        Return ribbonXML

    End Function